### PR TITLE
Add IE8 fixes (from @ etiquet)

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

*Please update this template with something that matches your PR*

### CODE REVIEW

**Comment by etiquet** [comment]
Hi guys this is crypto for me. Could you explain ? 
Many thanks.

**Comment by etiquet** [comment]
Hello Trasher ,

I did a pass on the IE css. correct few things since the main css has evolved.
The attached version is the corrected file.
[styles_ie.css.zip](https://github.com/glpi-project/glpi/files/888327/styles_ie.css.zip)

I tested fine against the glpi-master today with firefox (linux) / IE8 / IE11 
and with formcreator 0.9 1.3.4. ( will pursue with the latest version later on)  

Few  limitations :
- the create tickets interface in not usable. (has been the case previously -> use formcreator)
- the select2-container (jquery dropdown menu)  with images (like the css selector ) mix text and image. ( could not fix this is an acceptable regression )

Warning : the styles_ie.css is NOT included in the login page,
so no use to patch the login page..
will do the job I think.
Many thanks for working on this.
Eric

**Comment by trasher** [comment]
Hello,

I've included your new stylesheet. I also fix the inclusion on the login page (was still for IE <= 6; not 8).

**Comment by trasher** [comment]
@etiquet please test with current branch and tell us if everything is OK. Waiting your feedback to merge this one.

File: css/styles_ie.css [hunk 1]
```diff
@@ -30,11 +30,11 @@
 */
 /* ################--------------- GLPI CSS style MOD and HACK for IE  ---------------#################### */
 
+
```
Comment (etiquet) [inline css/styles_ie.css]: #logo_login{
   background: url(../pics/logos/logo-GLPI-250-white.png) center no-repeat ; 
}

**Pull request merged**